contributing_to_docs/doc_guidelines.adoc

```diff
@@ -400,11 +430,6 @@ When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syn
 == Writing text snippets
 A _text snippet_ is an optional component that lets you reuse content in multiple modules and assemblies. Text snippets are not a substitute for modules but instead are a more granular form of content reuse. While a module is content that a reader can understand on its own (like an article) or as part of a larger body of work (like an assembly), a text snippet is not self-contained and is not intended to be published or cross referenced on its own.
 
-[IMPORTANT]
-====
-Only include entire snippets in an assembly or module. Including link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-lines/[lines by content ranges] can lead to content errors when the included file is subsequently updated and is not permitted.
-====
-
```

Comment:
I would actually keep this note, since we still aren't technically allowing includes by line. Maybe just update it to say either include by tag or include full snippets?

Reply:
I didn't remove it, I moved it to a more relevant area and broadened it includes by line in general, actually.